classes/fields/number.php

```diff
@@ -246,6 +246,12 @@ public function regex( $value = null, $name = null, $options = null, $pod = null
 	 * {@inheritdoc}
 	 */
 	public function validate( $value, $name = null, $options = null, $fields = null, $pod = null, $id = null, $params = null ) {
+		$return = parent::validate( $value, $name, $options, $fields, $pod, $id, $params );
+
+		$errors = array();
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 305965969cd8f04e7c461f29cc57b017</sub>